### ISSUE TITLE
Return size from info is not with unit "byte"

### ISSUE THREAD

**Issue by WindzCUHK** [hacksparrow/node-easyimage#62]
The return value of file size from identify has a unit with it.

Simply using the following will return a wrong value
info.size    = parseInt(temp[4]);

**Comment by hacksparrow** [comment]
`parseInt()` will discard any non-number values after the leading numbers, so it does return the correct value in bytes.

Try this:
```
var val = '1024GIGABytes'
console.log(parseInt(val)) // 1024
```

**Comment by WindzCUHK** [comment]
sorry for my poor English...

What I mean is that, when I run the following command:

	$identify -format '%m %z %w %h %b %x %y %f' 3.png

I get:

	PNG 8 2880 1800 2.034MB 56 56 3.png

So the image size is 2.034MB.
However, what I get in **info.size** is **2** and it is not in the unit of byte,
i.e., 2MB > 2KB, but 2 < 2*1024...

I suggest using:

	function parseSize(sizeString) {
	    var unit = {
	        KB: 1024,
	        MB: 1048576,
	        GB: 1073741824
	    };

	    var rx = /^(\d\.\d)([KMG]B)$/;  // regex for extract the float value and its unit
	    var sizeArray = rx.exec(sizeString);
	
	    return parseFloat(sizeArray[1]) * unit[sizeArray[2]];
	};

Although this only return a approximation as imagemagick may round the value, at least it has a unit of byte.

**Comment by hacksparrow** [comment]
Oh, I got it. That's a valid issue. IM's output value sometimes differs wildly from version to version. Can you send a pr, if you can?

**Comment by WindzCUHK** [comment]
What is 'pr' mean? @@

if u mean IM version,
Version: ImageMagick 6.9.1-0 Q16 x86_64 2015-03-22 http://www.imagemagick.org
Copyright: Copyright (C) 1999-2015 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Features: DPC OpenCL
Delegates (built-in): bzlib fftw fontconfig freetype jng jpeg lcms lzma png tiff webp x xml zlib

and my OS is
Software  OS X 10.9.5 (13F1077)

**Comment by hacksparrow** [comment]
PR means pull request. You clone the repo, make the changes you suggested and create a pull request so that your changes can be merged to the codebase.

**Comment by hacksparrow** [comment]
Thanks for the pr. It was failing for the B (byte) unit, fixed it, and published in 2.0.2.
